### PR TITLE
Add wagmi demo

### PR DESCRIPTION
**Pull request by taycaldwell**
### _Summary_

Update and add the wagmi code sample currently published on codesandbox.io to version control.

### _How did you test your changes?_

Manual testing.

`cd examples/<sample name>`
`npm install`
`npm start`

### CODE REVIEW

File: examples/wagmi-demo/.gitignore [hunk 1]
```diff
@@ -0,0 +1,21 @@
+# dependencies
```
Comment (cb-jake) [inline examples/wagmi-demo/.gitignore:1]: is this needed or can we update the root level gitignore?

File: examples/wagmi-demo/README.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
+# Coinbase Wallet: Integrate with wagmi
+
+This sample application demonstrates how to integrate Coinbase Wallet into a decentralized application using the wagmi library.
+
+Install packages:
+
+`npm install`
```
Comment (cb-jake) [inline examples/wagmi-demo/README.md:7]: nit: favor `yarn` over `npm` but include commands for both?

**Review by cb-jake — DISMISSED** [review]
lgtm

**Comment by cb-jake** [comment]
@taycaldwell can we close these example PRs if they're not needed?

**Comment by taycaldwell** [comment]
Oh sorry, I should probably merge these. Let me ensure they aren't stale first and we can review/merge.

**Comment by Hlllllllii** [comment]
Sent from my iPhoneOn 13 Dec Reiwa 4, at 12:46, taycaldwell ***@***.***> wrote:﻿
Oh sorry, I should probably merge these. Let me ensure they aren't stale first and we can review/merge.

**Comment by bangtoven** [comment]
merged into https://github.com/coinbase/coinbase-wallet-sdk/pull/796

**Pull request closed without merging**